compatibility/compatibility-tests/src/intTest/java/org/projectnessie/tools/compatibility/tests/AbstractCompatibilityTests.java

```diff
@@ -56,6 +57,7 @@
 import org.projectnessie.versioned.storage.common.persist.ObjId;
 
 @VersionCondition(maxVersion = Version.NOT_CURRENT_STRING)
+@Tag("nessie-multi-env")
```

Comment:
Are tags inherited from outer classes? I wish we could only put them where we have `
@ExtendWith(NessieUpgradesExtension.class)` or similar extensions.

Reply:
Can we add tags from the engine?

Reply:
> Are tags inherited from outer classes?

Did you mean "from parent classes"? (This particular class is not inner/nested.)

If so, yes, which is why I placed the tag here, instead of in the 2 child classes: `ITOlderClients` and `ITOlderServers`.

Do you prefer to have the tag on each concrete test class?

> Can we add tags from the engine?

Apparently not, see https://github.com/junit-team/junit5/issues/2134.

Reply:
> Do you prefer to have the tag on each concrete test class?
IMHO, it would be nice to add `@Tag("nessie-multi-env")` where we have `@ExtendWith(NessieUpgradesExtension.class)`. I suppose the intent will be clear to the reader.

It looks like `AbstractCompatibilityTests` is the only case where that is not so, so yes, I think it would be nice to move the tag from this class to its sub-classes: `ITOlderClients` and `ITOlderServers`. WDYT?